src/main/scala/com/webtrends/harness/component/cache/Cacheable.scala

```diff
@@ -186,10 +182,10 @@ trait Cacheable[T] extends Serializable {
     cacheRef.resolveOne onComplete {
       case Success(s) =>
         writeInCache(s, cacheKey)(timeout, executor) onComplete {
-          case Success(succ) => p success true
-          case Failure(f) => p success false
```

Comment:
I can't comment on the original intent, but I searched through all our projects and couldn't find any instance where we were paying attention to the success/failure of cache writes.

There will be a change in the cache read behavior though. Before this change, some read failures were treated as a Success(None) while others would hang until the future times out. There will be a corresponding wookiee-cache-memcache change to make sure we are handling the future failure cleanly.